src/js/controller/captions.js

```diff
@@ -238,13 +242,7 @@ define([
                 });
             } else {
                 status = utils.tryCatch(function() {
```

Comment:
do you think I could remove the trycatch? I took out the thrown error but im not sure if it may throw another unexpected one